Formula/jello.rb

```diff
@@ -0,0 +1,23 @@
+class Jello < Formula
```

Comment:
```suggestion
class Jello < Formula
  include Language::Python::Virtualenv
```
This should get the build to work.